pkg/events/events.go

```diff
@@ -130,6 +138,30 @@ func (e event) CorrelationIDs() []string                { return e.innerEvent.Co
 func (e event) Origin() string                          { return e.innerEvent.Origin }
 func (e event) Caller() string                          { return e.caller }
 func (e event) Visibility() *ttnpb.Rights               { return e.innerEvent.Visibility }
+func (e event) AuthType() string {
+	if e.innerEvent.Authentication != nil {
```

Comment:
nit; prefer indenting error flow and, hence, flipping this `if`(same below). See https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow